matminer/featurizers/structure.py

```diff
@@ -912,3 +913,33 @@ def get_op_stats_vector_diff(s1, s2, max_dr=0.2, ddr=0.01, ddist=0.01):
 
     return dr[idx], delta[idx]
 
+
+class EwaldEnergy(BaseFeaturizer):
+    """Compute the energy from Coulombic interactions
+
+    Note: The energy is computed using _charges already defined for the structure_.
+    Consider assigning them using the ``add_charges_*`` from ``pymatgen`` before calling this class
+
+    Features:
+        ewald_energy - Coulomb interaction energy of the structure"""
+
+    def __init__(self, accuracy=None):
+        """
+        Args:
+            accuracy (int): Accuracy of Ewald summation, number of decimal places
+        """
+        self.accuracy = accuracy
+
+    def featurize(self, strc):
+        # Compute the total energy
+        ewald = EwaldSummation(strc, acc_factor=self.accuracy)
+        return [ewald.total_energy]
+
+    def feature_labels(self):
+        return ("ewald_energy",)
+
+    def implementors(self):
+        return ("Logan Ward",)
+
+    def citations(self):
+        return []
```

Comment:
Can you cite Ewald and also pymatgen?

Ewald ref is:

@article{Ewald1921,
author = {Ewald, P. P.},
doi = {10.1002/andp.19213690304},
issn = {00033804},
journal = {Annalen der Physik},
number = {3},
pages = {253--287},
title = {{Die Berechnung optischer und elektrostatischer Gitterpotentiale}},
url = {http://doi.wiley.com/10.1002/andp.19213690304},
volume = {369},
year = {1921}
}